.gitignore

```diff
@@ -36,8 +41,9 @@ glean-core/data
 glean_app*
 !glean_app.c
 
-samples/ios/app/.venv/
 .mypy_cache/
 .coverage
 htmlcov/
 *.log
+
+Glean.framework.zip
```

Comment:
Let's just add `*.zip`. I don't think we ever have a use for adding zipped files ;)